GenericWorkflows/build-and-test.yml

```diff
@@ -1,20 +1,19 @@
-name: Build and Test
+name: Run Unit Test
 
 ####################################################################################################
 ##  WORKFLOW TRIGGER
 ####################################################################################################
 on:
-  # Workflow will run on any pull request to main.
+  # Workflow will run on pull requests to the main branch.
   pull_request:
-    branches:
-      - main
+    branches: [ main ]
 
 ####################################################################################################
 ##  WORKFLOW JOBS
 ####################################################################################################
 jobs:
-  # Calls the shared build-and-test workflow.
-  call_build_and_test:
-    name: Call build-and-test workflow
-    uses: RockLib/RockLib.Workflows/.github/workflows/build-and-test.yml@main
+  # Calls the shared unit-test workflow.
```

Comment:
Why was this renamed when the unit-test.yml still builds and tests?

Reply:
Since there's now two separate workflow files that run as part of PR testing I thought it made sense to use similar names with `unit-test` and `snyk-test` since that's the primary purpose of those two workflows.

Reply:
Oh ok I thought it persisted the build for publishing.  But it rebuilds.  Carry on.